nltk/test/unit/test_tokenize.py

```diff
@@ -14,24 +14,20 @@
     LegalitySyllableTokenizer,
 )
 
-def setup_module(module):
-    import pytest
 
+def load_stanford_segmenter():
     try:
         seg = StanfordSegmenter()
         seg.default_config("ar")
         seg.default_config("zh")
-    except LookupError as e:
-        pytest.skip(
-            "Tests for nltk.tokenize.stanford_segmenter skipped: %s" % str(e)
-        )
-
-    try:
-        StanfordTokenizer()
+        return True
     except LookupError:
-        pytest.skip(
-            "Tests for nltk.tokenize.stanford are skipped because the stanford postagger jar doesn't exist"
-        )
+        return False
+
+
+stanford_segmenter = pytest.mark.skipif(
```

Comment:
I would suggest naming this `check_stanford_segmenter`, that makes it clearer what it does.

Reply:
That's a great idea. The current name is quite confusing. I've made the change, thank you for the suggestion.